classes/05class/exercises/c05-3m01/alanlima/Makefile

```diff
@@ -0,0 +1,30 @@
+RUNNER ?= docker-compose run --rm awscli
+BUCKET_NAME?=
+
+create_bucket:
+	@$(RUNNER) make _create_bucket
+.PHONY:create_bucket
+
+delete_bucket:
+	@$(RUNNER) make _delete_bucket
+.PHONY:delete_bucket
+
+list_bucket_objects:
+	@$(RUNNER) make _list_bucket_objects
```

Comment:
always going beyond, good work :)

Reply:
Thanks, I like to play around with the cli. I missed it 😄